src/rgw/rgw_rados.cc

```diff
@@ -4847,6 +4888,9 @@ int RGWRados::init_watch()
   notify_oids = new string[num_watchers];
   watchers = new RGWWatcher *[num_watchers];
 
+  int error = 0;
+  vector<librados::AioCompletion *> completions;
```

Comment:
unused local variable?

Reply:
@cbodley it's actually being used, see below

Reply:
i don't see `completions` below - it looks like the pointer moved into `RGWWatcher`

Reply:
@cbodley yeah, you're right ... I was looking at `error`, and missed `completions`.